app/lib/screens/EditorScreen/Editor.dart

```diff
@@ -0,0 +1,320 @@
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:html_editor_enhanced/html_editor.dart';
+
+void main() => runApp(const HtmlEditorExampleApp());
+
+var arr = [
+  Colors.grey,
+  Colors.purple[300],
+  Colors.pink[100],
+  Colors.teal[400],
+  Colors.purple[700],
+  Colors.orange,
+  Colors.blue[300],
+  Colors.green,
+  Colors.yellow[100],
+  Colors.yellow,
+  Colors.pink,
+  Colors.red
+];
+
+List<String> _tagList = [];
+
+class HtmlEditorExampleApp extends StatelessWidget {
+  const HtmlEditorExampleApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Editor',
+      theme: ThemeData.dark(),
+      darkTheme: ThemeData.dark(),
+      home: const HtmlEditorExample(title: 'Create FAQ'),
+    );
+  }
+}
+
+class HtmlEditorExample extends StatefulWidget {
+  const HtmlEditorExample({Key? key, required this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  _HtmlEditorExampleState createState() => _HtmlEditorExampleState();
+}
+
+class _HtmlEditorExampleState extends State<HtmlEditorExample> {
+  String result = '';
+  OverlayEntry? entry;
+  TextEditingController tagTitleController = TextEditingController();
+
+  final HtmlEditorController controller = HtmlEditorController();
+
+  @override
+  void initState() {
+    super.initState();
+    _tagList.clear();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () {
+        if (!kIsWeb) {
+          controller.clearFocus();
+        }
+      },
+      child: Scaffold(
+        appBar: AppBar(
+          backgroundColor: AppColors.backgroundColor,
+          leading: TextButton(
+            onPressed: () {
+              Navigator.pop(context);
+            },
+            child: const Icon(
+              Icons.close,
+              color: Colors.white,
+            ),
+          ),
+          title: Text(widget.title),
+          centerTitle: true,
+          elevation: 0,
+          actions: [
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: ElevatedButton(
+                style: ElevatedButton.styleFrom(
+                  primary: AppColors.primaryColor,
+                  shape: const StadiumBorder(),
+                ),
+                onPressed: () {
+                  showBottomSheet();
+                },
+                child: const Text('Save'),
+              ),
+            )
+          ],
+        ),
+        body: SingleChildScrollView(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: <Widget>[
+              HtmlEditor(
+                controller: controller,
+                htmlEditorOptions: const HtmlEditorOptions(
+                  hint: 'Write about the FAQ',
+                  shouldEnsureVisible: true,
+                ),
+                htmlToolbarOptions: HtmlToolbarOptions(
```

Comment:
this disappears as soon as I enter the editor (and the keyboard appears), rendering this completely useless. Did this ever work for you? Could you make things bold?

![Screenshot_1652475720](https://user-images.githubusercontent.com/40496/168389153-6aea39b0-6a5e-4c1a-b862-fe51eceb5b8b.png)

Reply:
though, it seems to work sometimes..

Reply:
this is still a problem.

Reply:
It worked for me always. never missed